charts/ratify/templates/configmap.yaml

```diff
@@ -54,14 +54,14 @@ data:
                 "name":"notaryv2",
                 "artifactTypes" : "application/vnd.cncf.notary.v2.signature",
                 "verificationCerts": [
-                    "/usr/local/ratify-certs"
+                    "/usr/local/ratify-certs/notary"
                   ]
             {{- if .Values.cosign.enabled }}
             },
             {
                 "name": "cosign",
                 "artifactTypes": "org.sigstore.cosign.v1",
```

Comment:
There is some inconsistency here on cosign Artifact type, the walk through shows "vnd.dev.cosign.simplesigning.v1+json"
 , https://github.com/deislabs/ratify/pull/110/files

Reply:
i can see cosign doc showing "vnd.dev.cosign.simplesigning.v1+json" as the mediaType https://github.com/sigstore/cosign/blob/8dc365ae43931c8695f4ffab164e62395e108926/USAGE.md

Reply:
update walkthrough to "org.sigstore.cosign.v1"

Reply:
Could you help point to the doc for `org.sigstore.cosign.v1`?

Reply:
docs/examples/ratify-on-aws.md was updated to org.sigstore.cosign.v1, clarified with Eric during the community meeting, this string it something we came up so Ratify could verifiers